[2022-08-09 12:52:42,798] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: parallel_dag.load_a manual__2022-08-09T12:52:31.325160+00:00 [queued]>
[2022-08-09 12:52:42,804] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: parallel_dag.load_a manual__2022-08-09T12:52:31.325160+00:00 [queued]>
[2022-08-09 12:52:42,805] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-08-09 12:52:42,806] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-08-09 12:52:42,806] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-08-09 12:52:42,815] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): load_a> on 2022-08-09 12:52:31.325160+00:00
[2022-08-09 12:52:42,818] {standard_task_runner.py:52} INFO - Started process 1096 to run task
[2022-08-09 12:52:42,821] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'parallel_dag', 'load_a', 'manual__2022-08-09T12:52:31.325160+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp6j8h4jqd', '--error-file', '/tmp/tmpk4b8py5s']
[2022-08-09 12:52:42,822] {standard_task_runner.py:80} INFO - Job 46: Subtask load_a
[2022-08-09 12:52:42,858] {task_command.py:369} INFO - Running <TaskInstance: parallel_dag.load_a manual__2022-08-09T12:52:31.325160+00:00 [running]> on host 54b556b4c00a
[2022-08-09 12:52:42,901] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=load_a
AIRFLOW_CTX_EXECUTION_DATE=2022-08-09T12:52:31.325160+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-09T12:52:31.325160+00:00
[2022-08-09 12:52:42,902] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-09 12:52:42,903] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 10']
[2022-08-09 12:52:42,910] {subprocess.py:85} INFO - Output:
[2022-08-09 12:52:52,913] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-09 12:52:52,981] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=load_a, execution_date=20220809T125231, start_date=20220809T125242, end_date=20220809T125252
[2022-08-09 12:52:53,015] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-09 12:52:53,030] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
